airbyte-webapp/src/components/ColumnSortButton/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { FormattedMessage } from "react-intl";
+
+import styles from "./ColumnSortButton.module.scss";
+import SortIcon from "./SortIcon";
+
+interface ColumnSortButtonProps {
+  onClick: () => void;
+  formattedMessageId: string;
+  wasActive: boolean;
+  lowToLarge: boolean;
+}
+
+export const ColumnSortButton: React.FC<ColumnSortButtonProps> = ({
```

Comment:
This is not just the button but also contains the text of the heading, the name should be `SortableTableHeader`. This should also be part of the Table export, so it's probably best if it goes into `components/ui/Table`